aiohttp_debugtoolbar/middlewares.py

```diff
@@ -103,7 +103,9 @@ def toolbar_middleware(request):
                 #
                 # response = request.invoke_subrequest(subrequest)
                 body = tb.render_full(request).encode('utf-8', 'replace')
-                response = web.Response(body=body, status=500)
+                response = web.Response(
+                    body=body, status=500,
+                    content_type='text/html; charset=utf-8')
```

Comment:
The patch is wrong, sorry.

`content_type` parameter is for type only, not charset.
Ctor adds `; charset=utf-8` to given parameter ultimately: https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/web_reqrep.py#L742-L748

`content_type='text/html'` should be enough.

Perhaps we need *charset* param for `aiohttp.web.Response` ctor, I'll file the issue.